src/components/Tooltip/types.ts

```diff
@@ -25,16 +27,59 @@ type TooltipProps = ChildrenProps & {
     /** Unique key of renderTooltipContent to rerender the tooltip when one of the key changes */
     renderTooltipContentKey?: string[];
 
+    /** Whether to left align the tooltip relative to wrapped component */
+    shouldForceRenderingLeft?: boolean;
+
+    shouldForceRenderingBelow?: boolean;
+
+    /** Additional styles for tooltip wrapper view */
+    wrapperStyle?: StyleProp<ViewStyle>;
+};
+
+type TooltipRect = {
```

Comment:
Can't we actually "borrow" the `LayoutRectangle` type? Isn't it in the React Native core? TypeScript should be fine even with assigning `DomRect` to `LayoutRectangle`, I think